packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -523,16 +524,23 @@ export const GiantTrackTile = ({
             </div>
             <div className={styles.artistWrapper}>
               <div className={cn(fadeIn)}>
-                <span>By</span>
+                <span>By </span>
```

Comment:
why was this space necessary? is it possible this will cause other hover issues, etc? probably fine but worth thinking about a little which maybe you have

Reply:
there was a paddingLeft on the link to simulate a space, so removed that and added space here :)